src/components/content-header-card/content-header-card.module.css

```diff
@@ -0,0 +1,161 @@
+.contentHeaderCard {
+	padding: 0;
+
+	& .cardTop {
```

Comment:
This could be a personal preference or i'm not sure we've officially discussed this, but for the most part in developer we tend to define component classes at the root level, with some exceptions. I find this makes the CSS easier to read and everything is already scoped to the component. I'd be curious to hear your take though on the benefits of nesting.

Reply:
Hah, I find the nesting easier to read because it follows the same hierarchical shape of the HTML.  🥴 I feel like there's definitely a preference aspect to this, but I do see the value in all aligning.

Materially, I like the flexibility of being able to write scoped logic (e.g.; I can have two `left` classes in my component and contextually style them based off of them being nested in a component).  Additionally I can write styles that are contextual to a class being in a specific context.

Reply:
Resurfacing what we chatted about via slack. The expectation is that classes are defined at the root of the file unless the nesting is actually needed, like in the case of the flex positioning. If you have multiple classes named `left`, my suggestion is to adjust the names to be more descriptive.

Reply:
Fwiw this isn't necessarily blocking feedback, if you don't want to refactor this whole file. But I do think we should get aligned for future work.

Reply:
I'm going to defer refactoring this for now; and will take this as feedback for future components I will avoid doing this to align with the team's approach!